src/app/token-options/mixins/token-option-mixin.ts

```diff
@@ -78,22 +78,22 @@ export function TokenOptionMixin<TBase extends Constructor<IGridViewDataSource>>
         constructor(...args: any[]) {
             super(...args);
             this.registerDataPointSource(() => ({
-                colName: 'Group',
+                colName: 'Token Option Group',
                 type: 'string',
                 value: this.group.name
             }));
             this.registerDataPointSource(() => ({
-                colName: 'Type',
+                colName: 'Token Option Type',
                 type: 'string',
                 value: this.type
             }));
             this.registerDataPointSource(() => ({
-                colName: 'ID',
+                colName: 'Token Option ID',
                 type: 'number',
                 value: this.id
             }));
             this.registerDataPointSource(() => ({
-                colName: 'Name',
+                colName: 'Request’s Name',
```

Comment:
Since the grid view is created for instructors (at least for now), I think we should still use "Token Option Name" here to avoid confusion about the concept of request and token options.